feature_engine/outliers/__init__.py

```diff
@@ -6,4 +6,5 @@
 from .winsorizer import Winsorizer
 from .trimmer import OutlierTrimmer
 
+
```

Comment:
Yea I was quite confused by the amount of files that were included in the PR (I'm a git noobie) <br> I think all those changes are a result of some lint/typecheck/stylecheck call, I never edited all those files myself. Then they somehow got staged together with the rest. I'll try and fix that

Reply:
Yes, I thought so. It doesn't matter. We can remove them. I tend to do `git add file1.py file2.py` instead of `git add .` to avoid committing unwanted files